framework/src/modules/token/method.ts

```diff
@@ -706,8 +742,37 @@ export class TokenMethod extends BaseMethod {
 		this.events.get(TokenIDSupportedEvent).log(methodContext, tokenID);
 	}
 
-	public async removeSupportTokenID(methodContext: MethodContext, tokenID: Buffer): Promise<void> {
-		await this.stores.get(SupportedTokensStore).removeSupportForToken(methodContext, tokenID);
+	public async removeSupport(methodContext: MethodContext, tokenID: Buffer): Promise<void> {
+		const [chainID] = splitTokenID(tokenID);
+
+		const allSupportedTokenEntries = await this._isAllTokensSupported(methodContext);
+
+		if (allSupportedTokenEntries) {
+			throw new Error('All tokens are supported.');
+		}
+
+		if (tokenID.equals(this.getMainchainTokenID()) || chainID.equals(this._config.ownChainID)) {
+			throw new Error('Cannot remove support for the specified token.');
+		}
+
+		const supportedTokens = await this._getSupportedTokens(methodContext, chainID);
```

Comment:
this return value is not boolean

Reply:
Is it supposed to be a boolean?

Reply:
`supportedTokens(chainID)` is just a notation, not an actual variable. It is used to make the LIP more readable. 

![image](https://github.com/LiskHQ/lisk-sdk/assets/48478060/12d9d808-9eab-4310-adcc-597850102c32)